### ISSUE TITLE
Turing Machine not calibrating, locks to pitch

### ISSUE THREAD

**Issue by skramsino** [TomWhitwell/TuringMachine#134]
Hi,

having tried various square waves to calibrate, I can´t stabilise the output.
When controlling pitch in an oscillator, randomisation "stops" at the highest pitch and circles around in quartertones. So, first theres´s a random sequence of about 4-6 notes, then it freezes at around the highest area.
Here are pictures of the pcb.

![IMG_6768](https://user-images.githubusercontent.com/65658101/82804601-db46aa80-9e82-11ea-84f8-5c3ded336105.jpg)
![IMG_6769](https://user-images.githubusercontent.com/65658101/82804618-e00b5e80-9e82-11ea-8875-a19165c6b564.jpg)
![IMG_6770](https://user-images.githubusercontent.com/65658101/82804621-e13c8b80-9e82-11ea-8e4f-04a942aa07c9.jpg)
![IMG_6771](https://user-images.githubusercontent.com/65658101/82804623-e1d52200-9e82-11ea-9eaf-0133eaf9fd5d.jpg)

**Comment by katspaugh** [comment]
Any luck fixing it, @skramsino?

**Comment by skramsino** [comment]
> 

No.. gave up. Never found the mis-soldering. Will build a new one!
